internal/vm.h

```diff
@@ -83,6 +83,23 @@ void rb_check_stack_overflow(void);
 extern uint64_t rb_vm_insns_count;
 #endif
 
+extern bool rb_free_on_shutdown;
+
+void rb_free_default_rand_key(void);
+void rb_free_encoded_insn_data(void);
+void rb_free_environ(void);
+void rb_free_global_enc_table(void);
+// rb_free_loaded_builtin_table needs a week definition in case minibuiltin.c is not used
+// this is required for the wasm build.
+void rb_free_loaded_builtin_table() __attribute__((weak));
```

Comment:
Looks like `__attribute__((weak))` is not supported in Windows, but removing this break the wasm build.

Maybe there is another builtin.c equivalent place I need to define this so that wasm has it and Windows is happy?

Reply:
@mame It seems fine.
Just one thing, don't use pre-ANSI-style declaration.